style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Quicksand", sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  background-position: 50% 50%;
+  background-size: cover;
+  height: 100vh;
+  color: rgb(0, 0, 0);
+}
+
+#time {
+  font-size: 8rem;
+}
+
+h1 {
+  margin-bottom: 3rem;
+}
+
+h2 {
+  margin-bottom: 0.5rem;
+  opacity: 0.6;
+}
+
+@media (max-width: 700px) {
+  #time {
+    font-size: 6rem;
+  }
+}
```

Comment:
Markup must be adaptive.
Check out the background picture.  On the screenshot you can notice that it is repeated (see below).

![image](https://user-images.githubusercontent.com/45519864/111122452-200d1080-857f-11eb-98ea-c914c183851a.png)

Reply:
DONE!